src/Features/CSharp/Portable/Completion/CompletionProviders/ExplicitInterfaceMemberCompletionProvider.cs

```diff
@@ -83,7 +85,7 @@ public override async Task ProvideCompletionsAsync(CompletionContext context)
                     return;
                 }
 
-                var members = symbol.GetMembers();
+                var members = symbol.GetMembers().WhereAsArray(m => !IsInvalidCompletionMethodKind(m));
```

Comment:
Given we're doing a loop through members later, should we just filter those in that loop instead of having the intermediate allocation?

Reply:
Yea that definitely makes sense. It also is a little more straightforward to read since it avoids using negation.